src/sonic-config-engine/sonic-cfggen

```diff
@@ -181,6 +190,8 @@ def main():
     group.add_argument("-w", "--write-to-db", help="write config into configdb", action='store_true')
     group.add_argument("--print-data", help="print all data", action='store_true')
     group.add_argument("--preset", help="generate sample configuration from a preset template", choices=get_available_config())
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument("-f", "--find", help="Find/Search for a specific key")
```

Comment:
what is the difference between --var-json.

Reply:
--var-json is used to query one level higher keys. For example: BGP_NEIGHBOR, DEVICE_METADATA, INTERFACE, PORT, etc.. where as --find is used query one level down. For example Ethernet0 under PORT.

Reply:
then, why not use -k, --key option which is more specific?

Reply:
I though so as well, since -k is taken by hwSKU, I used -f. Please suggest if you have any other better alternative.

Reply:
-K

Reply:
Sounds good. Let me change that!